### ISSUE TITLE
Starting a private conversation should be easier

### ISSUE THREAD

**Issue by Yoric** [vector-im/element-web#15709]
Right now, if I wish to start a private conversation with someone I have two possibilities that I know of:

1. Click on `People +`.
2. Pick the person I wish to chat with.
3. This opens a private conversation.

OR

1. Click on `Room Info`.
2. Click on user name (which may require several steps in a room with 100+ users).
3. Click on `Direct Message`.
4. This opens a private conversation.

I would like the following

1. Click (or right click, or double-click, or long-click...) on the user in the current conversation.
2. This opens a private conversation.

alternatively

1. Click on the existing `...` menu at the end of a line of my current conversation.
2. Pick option "Reply in private".
3. This opens a private conversation.

**Comment by t3chguy** [comment]
You can click on a user's avatar on their message then click direct message.

**Comment by Yoric** [comment]
You're right, there is a faster path than what I had noticed. I was trying to click on the user's name, rather than the avatar.

**Comment by uhoreg** [comment]
Even though it is possible to click on a user's avatar to get to the user's information and start a DM from there, it isn't very discoverable, so it still may be worth it for the design team to see if there's a better way.
